### ISSUE TITLE
Add meta information to client

### ISSUE THREAD

**Issue by bencooper222** [VandyHacks/vaken#22]
We need meta tags for Facebook and the like. We should also have a .webmanifest for icons and such. 

Presumably we want this to be configurable - need to figure out how to do this well.

**Comment by cktang88** [comment]
If configurability is hard, we can just have instructions telling them to just directly edit the html.

**Comment by aadibajpai** [comment]
> We need meta tags for Facebook and the like. We should also have a .webmanifest for icons and such.
> 
> Presumably we want this to be configurable - need to figure out how to do this well.

I feel like since we're using HtmlWebpackPlugin, we can just add meta tags there. Better than adding them directly in the html and the plugin part would need to be modified anyway for favicon now.

**Comment by bencooper222** [comment]
Yep, good point. I bet there's also a plugin that can do stuff like take one image and set `og:image` and `twitter:image` etc
